roles/tarsnap/tasks/main.yml

```diff
@@ -2,14 +2,15 @@
 # Tarsnap - backup
 
 - name: Copy key file into place
-  copy: src="keys/{{ inventory_hostname }}.key" dest=/root/tarsnap.key owner=root group=root mode=0600
+  template: src="keys/{{ inventory_hostname }}.key" dest=/root/tarsnap.key owner=root group=root mode=0600
```

Comment:
Vault can encrypt vars files but not `files/whartever`, so everyting is now moved to a variable